frontend-react/src/components/ManagePublicKey/ManagePublicKey.tsx

```diff
@@ -82,50 +91,77 @@ function ManagePublicKeySwitchDisplay() {
         }
     };
 
+    const showPublicKeyConfigured =
+        sender.length > 0 &&
```

Comment:
Could be variable overload, but I feel like it'd help to add some variable aliases for some of these pieces.  Like `sender.length > 0` and `(sender.length > 0 && !fileSubmitted && !hasPublicKey)` 🤷 

Might also be a good idea to infer an index in the steps based on the current state.  I don't think we should be passing arbitrary numbers to set the index, but we could do something like [what we're doing in FileHandler](https://github.com/CDCgov/prime-reportstream/blob/master/frontend-react/src/components/FileHandlers/FileHandler.tsx#L27), i.e., setting up an array of steps and inferring the index from the state like we discussed in a previous pull request.  Doesn't have to be part of this change, but I can imagine that we'll want to reuse this stepwise logic in the future so it'd be good to get a sense of what its capabilities should be.